### PR TITLE
Making the encoding consistent- changing lowercase utf to uppercase

### PR DESCRIPTION
**Pull request by charvimehradu**
Pull Request for Issue #39927 .

### Summary of Changes

- This PR also replaces the wrong Windows line endings (CR+LF) with correct Unix line endings (LF) in 2 xml files "administrator/modules/mod_loginsupport/mod_loginsupport.xml" and "installation/language/ru-RU/langmetadata.xml".

- Replacing lowercase utf with uppercase in .xml files.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
 		<exec executable="phpcs">
 			<arg value="--report=checkstyle" />
 			<arg value="--extensions=php" />
-			<arg value="--encoding=utf-8" />
+			<arg value="--encoding=UTF-8" />
```
Comment (joomdonation) [inline build.xml]: Maybe you should revert this back to utf-8 (I have no clue how it is used here, but it is not something we wanted to do from the original discussion).

**Comment by joomdonation** [comment]
Other than the small change which I mentioned, this look good to me. If you made the change, I will submit my test result.

File: administrator/components/com_associations/access.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (sandewt) [inline administrator/components/com_associations/access.xml]: Here and elsewhere and in some other places there is a space right before the closing tag.

`<?xml version="1.0" encoding="UTF-8" ?>`

Probably this is better (without a space).

`<?xml version="1.0" encoding="UTF-8"?>`

File: build.xml (hunk 1 above)
Comment (richard67) [inline build.xml]:
```suggestion
			<arg value="--encoding=utf-8" />
```
I agree. Please revert this change or accept my code change suggestions which will do the same.

Comment (charvimehradu) [reply build.xml]: done

File: administrator/modules/mod_loginsupport/mod_loginsupport.xml [hunk 3]
```diff
@@ -63,16 +63,16 @@
...
+				<field
+					name="automatic_title"
+					type="radio"
+					label="COM_MODULES_FIELD_AUTOMATIC_TITLE_LABEL"
+					layout="joomla.form.field.radio.switcher"
+					default="1"
+					filter="integer"
+					>
+					<option value="0">JNO</option>
+					<option value="1">JYES</option>
```
Comment (richard67) [inline administrator/modules/mod_loginsupport/mod_loginsupport.xml:75]: This difference here is shown because this PR replaces wrong Windows line endings (CR+LF) by correct Unix line endings (LF). But that's not really visible on GitHub. So your change is good, I just comment so other readers get notice, and maybe you should mention this in the description (initial comment) of this PR.

File: installation/language/ru-RU/langmetadata.xml [hunk 4]
```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
 <metafile client="installation">
 	<name>Russian (Russia)</name>
-	<version>4.2.2</version>
+	<version>4.2.2</version>
```
Comment (richard67) [inline installation/language/ru-RU/langmetadata.xml:4]: This difference here is shown because this PR replaces wrong Windows line endings (CR+LF) by correct Unix line endings (LF). But that's not really visible on GitHub. So your change is good, I just comment so other readers get notice, and maybe you should mention this in the description (initial comment) of this PR.

File: installation/language/ru-RU/langmetadata.xml [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
 	<creationDate>2022-09</creationDate>
 	<author>Russian Translation Team</author>
-	<copyright>(C) 2005 Open Source Matters, Inc.</copyright>
+	<copyright>(C) 2005 Open Source Matters, Inc.</copyright>
```
Comment (richard67) [inline installation/language/ru-RU/langmetadata.xml:7]: This difference here is shown because this PR replaces wrong Windows line endings (CR+LF) by correct Unix line endings (LF). But that's not really visible on GitHub. So your change is good, I just comment so other readers get notice, and maybe you should mention this in the description (initial comment) of this PR.

File: administrator/components/com_associations/access.xml (hunk 2 above)
Comment (richard67) [reply administrator/components/com_associations/access.xml]: @sandewt I'd prefer this to be done with another PR. One PR for one issue makes it easier. But if @charvimehradu wants to fix that with this PR here I am also ok with it.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on e073ecf490d69b1ecb5e1c0737ce3bf2c7008228<br /><br />I can confirm that this PR here catches all 584 cases which I can find on a clean 4.3-dev branch on Linux when doing a `find ./ -type f -name "*\.xml" -exec grep -Hn "encoding=\"utf-8\"" {} \; | grep "xml version="` and counting them with a `| wc -l`.

I've reviewed all changes and and can confirm they are correct.

Additional changes shown on GitHub for 2 xml files "administrator/modules/mod_loginsupport/mod_loginsupport.xml" and "installation/language/ru-RU/langmetadata.xml" but where you can't see what was changed are due to wrong Windows line endings CR+LF being replaced with correct Unix line endings LF.

Finally, redundant spaces have been removed from the XML tag. Ok by review. 

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39951">issues.joomla.org/tracker/joomla-cms/39951</a>.</sub>

**Comment by richard67** [comment]
I have not tested this item.<br /><br />Sorry, the last commit was not ok. Review comment will follow.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39951">issues.joomla.org/tracker/joomla-cms/39951</a>.</sub>

File: administrator/components/com_associations/access.xml (hunk 2 above)
Comment (charvimehradu) [reply administrator/components/com_associations/access.xml]: I fixed it

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on e073ecf490d69b1ecb5e1c0737ce3bf2c7008228<br /><br />All ok.

I have tested this item :white_check_mark: successfully on e073ecf490d69b1ecb5e1c0737ce3bf2c7008228<br /><br />I can confirm that this PR here catches all 584 cases which I can find on a clean 4.3-dev branch on Linux when doing a `find ./ -type f -name "*\.xml" -exec grep -Hn "encoding=\"utf-8\"" {} \; | grep "xml version="` and counting them with a `| wc -l`.

I've reviewed all changes and and can confirm they are correct.

Additional changes shown on GitHub for 2 xml files "administrator/modules/mod_loginsupport/mod_loginsupport.xml" and "installation/language/ru-RU/langmetadata.xml" but where you can't see what was changed are due to wrong Windows line endings CR+LF being replaced with correct Unix line endings LF.

Finally, redundant spaces have been removed from the XML tag. Ok by review. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39951">issues.joomla.org/tracker/joomla-cms/39951</a>.</sub>

File: build.xml (hunk 1 above)
Comment (sandewt) [reply build.xml]: ~done ?~

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on a336da2375fb9cbfe2cef965522a5fb0ee571d37<br /><br />I did some search and confirm that UTF-8 (in upper case) is recommended/standard. I also reviewed the changes from all the files (two times) and it looks good.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39951">issues.joomla.org/tracker/joomla-cms/39951</a>.</sub>

**Comment by joomdonation** [comment]
RTC.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39951">issues.joomla.org/tracker/joomla-cms/39951</a>.</sub>

**Comment by sandewt** [comment]
Thks all

**Comment by obuisard** [comment]
Thank you Charvi @charvimehradu for the PR!

**Comment by charvimehradu** [comment]
Happy to help!
